### ISSUE TITLE
Editor Input Alerts are not said always

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#247577]
Testing #247333

Please see the attached video where I am switching among editor inputs. Alert is not mentioning about which input I am in sometimes when I am moving among them

https://github.com/user-attachments/assets/dbb1fb50-3c3f-4b77-b549-b3647f1e393a

**Comment by meganrogge** [comment]
@hediet any idea why this would happen? here's the implementation https://github.com/microsoft/vscode/pull/247156/files#diff-4b06886a9a1805016ed2daacc1f667900b50621e9cc5a388046deb3075fff4b3R189

**Comment by justschen** [comment]
+1, i'm seeing this as well

**Comment by hediet** [comment]
I cannot reproduce on windows.
If it happens again, please enable trace logging and check for `[trace] [Window] alertFocusedEditor` in the window log.

**Comment by meganrogge** [comment]
Now I'm not able to reproduce on mac. @justschen or @sandy081 did you notice any patterns here? Also, did it ever repro when using the keyboard only, not mouse click, to focus the editors?

**Comment by justschen** [comment]
did not try with keyboard, but i tried clicking in and around the conflicts in both left and right editors. sometimes it would happen, but sometimes it wouldn't

**Comment by meganrogge** [comment]
I cannot reproduce on mac and the logs all look correct. @justschen could you pls try and see what happens in the logs?

<img width="1433" alt="Image" src="https://github.com/user-attachments/assets/9d2f353c-e0bd-4be3-9c4f-42ac6f8c35cf" />
